src/styles/blocks/stars.scss

```
@@ -0,0 +1,20 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(../images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+}
+
+.stars--1 :nth-child(1),
```

```suggestion
.stars--1 :nth-child(-n + 1),
```
better to make consistent selector with other cases